src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_data_containers.h

```diff
@@ -1058,6 +1058,8 @@ namespace sirf {
     public:
 
         CoilSensitivitiesVector() : GadgetronImagesVector(){}
+		CoilSensitivitiesVector(MRAcquisitionData& ad, const bool coil_resolved=true) :
+			GadgetronImagesVector(ad, coil_resolved){}
```

Comment:
would it make more sense to have
```suggestion
		CoilSensitivitiesVector(MRAcquisitionData& ad) :
			GadgetronImagesVector(ad, /*coil_resolved=*/true){}
```

Reply:
done